README.md

```diff
@@ -2,391 +1,310 @@
-            securityMap.put("user_id", "$ANONYMIZED_USER_ID");
-            securityMap.put("domain", "localhost");
-
-            // Create some data for the questions api
-            JSONObject req = new JSONObject();
-            req.put("type", "local_practice");
-            req.put("state", "initial");
-
-            // Create a questions JSONArrray
-            JSONArray questions = new JSONArray();
-
-            // Create a question
-            JSONObject question = new JSONObject();
-            question.put("response_id", "60005");
-            question.put("type", "association");
-            question.put("stimulus", "Match the cities to the parent nation");
-
-            // Add stimulus list
-            JSONArray stimList = new JSONArray();
-            stimList.put(0, "London");
-            stimList.put(1, "Dublin");
-            stimList.put(2, "Paris");
-            stimList.put(3, "Sydney");
-            question.put("stimulus_list", stimList);
-
-            // Add possible responses
-            JSONArray possibleResp = new JSONArray();
-            possibleResp.put(0, "Australia");
-            possibleResp.put(1, "France");
-            possibleResp.put(2, "Ireland");
-            possibleResp.put(3, "England");
-            question.put("possible_responses", possibleResp);
-
-            // Add validation
-            JSONObject validation = new JSONObject();
-            JSONArray validResp = new JSONArray();
-            validResp.put(0, "England");
-            validResp.put(1, "Ireland");
-            validResp.put(2, "France");
-            validResp.put(3, "Australia");
-            validation.put("valid_responses", validResp);
-            question.put("validation", validation);
-
-            // Add questions to questions array
-            questions.put(0, question);
-
-            // Finally add questions to request
-            req.put("questions", questions);
-
-
-            // Instantiate the SDK Init class with your security and request data:
-            init = new Init("questions", securityMap, consumerSecret, req);
-
-            // Call the generate() method to retrieve a JavaScript object
-            String questionJson = init.generate();
-
-            // Pass the object to the initialisation of any Learnosity API
-            // For instance in your jsp file you can have:
-            // <script src="//questions.learnosity.com"></script>
-            // var questionsApp = LearnosityApp.init(questionJson);
-
-            System.out.println(questionJson);
-        }catch (Exception e) {
-            System.out.println(e.getMessage());
-        }
-    }
-}
-```
+### **Development install from a Git clone**
+To install from the terminal, run this command:
 
-Raw JSON initialisation snippet:
+    git clone git@github.com:Learnosity/learnosity-sdk-Java.git
 
-```java
-import learnositysdk.request.Init;
+### **Alternative method 1: download the zip file**
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 0; Below](https://app.codacy.com/gh/Learnosity/learnosity-sdk-java/pullRequest?prid=8556170)
```suggestion
### **Alternative method 1: download the zip file**

```